contracts/incentives/RewardProgram.sol

```diff
@@ -284,19 +285,23 @@ contract RewardProgram is
 
   function fundProgram(uint256 amount) external onlyOwner {
     require(_programData.rewardToken != address(0), "RP:E-405");
-    IERC20 token = ERC20(_programData.rewardToken);
-    token.safeTransferFrom(msg.sender, address(this), amount);
+    IERC20Detailed token = IERC20Detailed(_programData.rewardToken);
+    token.transferFrom(msg.sender, address(this), amount);
```

Comment:
this should be "safeTransferFrom"

Reply:
@robsecord 

safeTransferFrom is not in the base ERC20 standard. 

The token needs to implement the safe token library, and the IONX token does not implement it.